client-runtime/src/main/java/com/microsoft/rest/v3/annotations/UnexpectedResponseExceptionType.java

```diff
@@ -6,7 +6,7 @@
 
 package com.microsoft.rest.v3.annotations;
 
-import com.microsoft.rest.v3.RestException;
+import com.microsoft.rest.v3.http.rest.RestException;
```

Comment:
This existed before, but I was wondering why this it is called "UnexpectedResponseExceptionType" rather than just "UnexpectedResponseException" 🤔

Reply:
I will look into this - it seems like a reasonable suggestion.

Reply:
I could go either way on this. Shorter is normally better, and I don't believe the 'Type' adds any particular value to the name. Keen to get @anuchandy's feedback.

Reply:
Let's leave as is for now.